src/lib.c

```diff
@@ -246,19 +246,41 @@ int get_new_lines(char filename[]){
  * @param n_stores number of stores
  * @return returns stores in an array, as structs (store_s)
  */
-store_s* load_distances(char filename[], int n_stores) {
+store_s* load_distances(char filename[], int* n_stores, double user_lat, double user_lon, int radius) {
```

Comment:
Hvorfor ikke bare tage `coordinates_s` som et argument, i stedet for to doubles.

Reply:
Ændre functions navnet til load_coordinates?